generated/java/grpc-google-common-protos/build.gradle

```diff
@@ -6,9 +6,9 @@ buildscript {
 
 apply plugin: 'java'
 
-description = 'GRPC library for grpc-google-common-protos'
+description = 'GRPC library for grpc-google-common-protos (beta)'
 group = 'com.google.api.grpc'
-version = "1.12.0"
+version = '0.12.0-beta'
```

Comment:
I think a more natural version to use would be `1.13.0-pre1`

Reply:
ok!